### PR TITLE
csvTable function to insert tables from files

### PR DESCRIPTION
**Pull request by pireddag**
The csvTable function inserts in a TeXmacs document a table using the contents of a csv file

### CODE REVIEW

File: miscellanea/csvTable/my-init-texmacs.scm [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    >>>
+    (explicit-buttons ("or big-table environment" (display "message from insert
+table widget")))))
+
+;; Map t a b l e tab to either a message for the user, if the cursor is outside
+;; all of the environments where it is expected to work, or to the insertion of
+;; a table at the cursor position
+(kbd-map
+  ("t a b l e tab"
+   (if (or (tree-is? (cursor-tree) 'document)
```
Comment (woutersj) [inline miscellanea/csvTable/my-init-texmacs.scm:22]: This is probably better achieved with some kind of `(:mode ...)` statement.

**Comment by woutersj** [comment]
You could make this into a small plugin which loads your code on initialisation, so users don't have to modify `my-init-texmacs.scm`.

**Comment by pireddag** [comment]
> You could make this into a small plugin which loads your code on initialisation, so users don't have to modify `my-init-texmacs.scm`.

Let me work on both issues. It did not occur to me that the plugin code can contain a shortcut.

**Comment by pireddag** [comment]
Almost there.

**Comment by pireddag** [comment]
I have made chaos in my fork, I will restatr from scratch

**Pull request closed without merging**